_posts/2016-02-04-ombulabs-products.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "New OmbuLabs Products"
+date: 2016-02-04 18:35:00
+categories: ["Design"]
+author: "cecilia"
+---
+
+As part of Christmas presents for the Ombu Labs' team, we worked on a series of products. We want to share with you the process and its final result.
+
+<!--more-->
+
+### It all started with an idea
+
+We decided to make 3 products: a t-shirt, a notebook and a tote bag. As it happens with every product, it starts with an idea, then some sketches, its design and, finally, its creation.
+
+### Products with design
+
+For the notebook, we chose a quote by Otl Aicher’s: “Good Art Inspires, Good Design Motivates”. Here you can see part of the design process, with some tweaks and final adjustments.
+
+![Notebook Design Process](http://www.ombushop.com/blog/wp-content/uploads/2016/02/cuaderno-proceso.png)
+
+![OmbuLabs Notebook](http://www.ombushop.com/blog/wp-content/uploads/2016/02/cuaderno.png)
+
+For the t-shirt we chose a very well known quote that inspires us day by day at work: Done is Better than Perfect.
+
+![Done is Better than Perfect](http://www.ombushop.com/blog/wp-content/uploads/2016/01/lettering-2.png)
+
+![OmbuLabs t-shirt](http://www.ombushop.com/blog/wp-content/uploads/2016/01/remera.png)
+
+And last, but not least, our tote bag . Its design is inspired by our day-to-day work and methodology. We made some illustrations and then created a final composition.
+
+![Working at OmbuLabs](https://d13yacurqjgara.cloudfront.net/users/703924/screenshots/2427355/working_at_ombulabs.gif)
+
+![OmbuLabs Tote Bag](https://d13yacurqjgara.cloudfront.net/users/703924/screenshots/2450871/totebag.png)
+
+We are very proud and happy to share this with you. We believe this process can be applied to any kind of project. Soon, these products will be available for sale in our online store!
```

Comment:
@cecilia-ombulabs Also, could you talk a bit more about the process you followed? It'd be great if you could share some of the paper/pencil sketches that you made. Maybe you could add a new section for "Our process" after  `### It all started with an idea`